.github/workflows/prbuild.yaml

```diff
@@ -1,7 +1,7 @@
 name: Build and Test Pull Request
 
 # Trigger the workflow on push or pull request
-on: 
+on:
```

Comment:
nit: seems like this is all whitespace changes we could maybe remove most of them?